docs/app/pages/Components/Chips/examples/Format.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div>
+    <md-chips class="md-primary" v-model="clubs" md-placeholder="Add club..." :md-format="toUppercase">
+      <label>La Liga Clubs</label>
+      <div class="md-helper-text">Three uppercase letters</div>
+    </md-chips>
+    <md-chips class="md-primary" v-model="artists" md-placeholder="Add artist..." :md-format="formatName">
+      <label>Artists</label>
+      <div class="md-helper-text">Try insert `Eugène Ysaÿe` twice. The formatter will remove diacritics.</div>
```

Comment:
Is this message correct? `Try insert 'Eugène Ysaÿe' twice`. I think a good one could be:

Reply:
Yes, I expect developers see this:

![image](https://user-images.githubusercontent.com/29639463/34368036-c1a27624-eaeb-11e7-8ae9-79df9c29ddc8.png)